helm-chart-sources/logstream-workergroup/values.yaml

```diff
@@ -65,6 +65,7 @@ nodeSelector: {}
 #  topologySpreadConstraints: []
 
 service:
+  enable: true
```

Comment:
Can we add the description of this value to helm-chart-sources/logstream-workergroup/README.md

Reply:
Added config values to the readme in the logstream-workergroup as requested